examples/erc1155/lib.rs

```diff
@@ -792,7 +792,7 @@ mod erc1155 {
             // Note: All of these tests are from the context of the owner who is either allowing or
             // disallowing an operator to control their funds.
             set_sender(owner);
-            assert!(erc.is_approved_for_all(owner, operator) == false);
+            assert!(!erc.is_approved_for_all(owner, operator));
```

Comment:
I personally don't like this since it's hard to see the two `!` side by side

Reply:
Totally agree, the issue is that it's currently not possible to define global clippy lints (see https://github.com/rust-lang/cargo/issues/5034) and I think fragmentation of linting rules in each example would be a worse trade-off.

Reply:
I added a compromise to this PR: passing the flag `-A clippy::bool_assert_comparison` to all `clippy` invocations. This allows e.g. `assert_eq(…, false);`. 

It's kinda ugly and we should create a follow-up issue to replace this as soon as the clippy issue is fixed.

Reply:
I've removed the above hack now again, after thinking a bit more about it I can't bring myself to hack it this way. Feel free to create a follow-up to switch it back and add global lints, once it's possible.